### DISCUSSION TITLE
Please Please Please:  Choose @tag default to open the 3rd panel

### DISCUSSION THREAD

**Discussion by LegolasMirkwood** [vkbo/novelWriter#1785] [Ideas]
Right clicking and "follow" is not intuitive to open the 3rd panel.  Put a button, something, anything.  😂

**Comment by LegolasMirkwood** [comment]
I just think, I know.  Whenever I First see NovelWriter for the first time, it's obviously lopsided.  I have to reference the video, twice now, to figure out how to make the 3rd panel appear.  I love your work and that you're doing it.  I love writing.

**Comment by vkbo** [comment]
The viewer panel opens whenever you select "View Document" or middle mouse button click on a document in the project. That's what it's for.
